docs/conf/australia/2020/news/cfp-extend.rst

```diff
@@ -0,0 +1,29 @@
+:template: {{year}}/generic.html
+
+.. post:: August 27, 2020
+   :tags: australia-2020, cfp, tickets
+
+Call for Proposals Extended
+===========================
+
+Greetings fellow documentarians!
+
+The conference is now just under four months away, and we hope you’re already planning for this year's remote installment of Write the Docs Australia and India conference.
+
+New deadline to submit your proposal is 14th Sept
+--------------------------------------------------
+
+The `Call for Proposals <https://www.writethedocs.org/conf/australia/2020/cfp/>`_ is now extended until **14th Sept 2020 at Midnight AEST** – We added two extra weeks
+to the deadline because there are enough things to worry about this year, and we didn't want CFP dates to stress you unnecessarily!
```

Comment:
```suggestion
to the deadline because there are enough things to worry about this year, and we wanted to give as many of you the opportunity to prepare your submissions.
```